packages/FormSync/FormSyncElement.js

```diff
@@ -157,7 +159,7 @@
         setupOriginalWatcher() {
             this.originalElement.addEventListener('change', () => {
                 // Don't auto submit if original form changed
-                this.clonedElement[this.property] = this.originalElement[this.property];
+                this.clonedElement[this.clonedProperty] = this.originalElement[this.property];
```

Comment:
Grad bemerkt: Können wir diese Zeile ev. ersetzen durch L184: `this.syncOriginalToCloned()`?